sprucelabsai/workspace.sprucebot-skills-kit
# Fix dialog resize bug

@sprucelabsai/engineers

## Description 
Fixes iframe not resizing when dialog hides.

## Type

## Steps to Test or Reproduce
Open a large dialog on the booking/owner settings and close it. Iframe should resize.
